Dockerfile

```diff
@@ -3,26 +3,12 @@ FROM python:3.6
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-RUN apt-get update && apt-get install -y libssl-dev
-RUN apt-get update && apt-get install -y python3-pip
-RUN apt-get update && apt-get install -y pandoc
-
-#Copy the application py-evm to the /usr/src/app folder
 COPY . /usr/src/app
 
-# Remove existing virtualenv directory
-RUN rm -rf venv_docker_build
-# Install python dependencies
-RUN pip install virtualenv
```

Comment:
I'm far out of my comfort zone regarding judging of ditching `virtualenv` is appropriate or not. @carver @pipermerriam any thoughts on this? Can we assume that a virtual directory isn't needed when we are based on a fresh system (Debian I think that is)

Reply:
Ok, judging by what web3 does, it really seems that a virtualenv can be ditched here:

https://github.com/ethereum/web3.py/blob/d436b0a799d92244ac8a419bf6da03f32a4feb4f/Dockerfile#L8

Reply:
Yeah, I think the premise of docker allows us to treat the main system python as a sort-of venv.